recipes/pyjc/meta.yaml

```diff
@@ -0,0 +1,35 @@
+package:
+  name: pyjc
+  version: "0.1.0"
+
+source:
+  fn: pyjc-0.1.0.tar.gz
+  url: https://pypi.python.org/packages/af/9d/6a789ddbd67caf159bc548c0e99b10bcf36e7e990d3097dba0b8cb4c62f5/pyjc-0.1.0.tar.gz
+  md5: f0ee545672b71f7a048d588c2bff96e6
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  skip: True  # [not (py36 and osx-64)]
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+
+test:
+  imports:
+    - pyjc
+
+about:
+  home: http://github.com/Atlas7/pyjc
+  summary: 'An educational deep learning framework'
+  license_family: MIT
+  license: MIT
```

Comment:
To actually include the license, please use the following syntax for the `license_file`:

https://conda-forge.org/docs/meta.html#packaging-the-license-manually

although it looks like license file is now in the `MANIFEST.in` so next release you can change it to `license_file: LICENSE.txt`

Reply:
Thanks for the link and all the suggestions! To be honest I am a bit stuck on this part (regarding `licence_file` attribute).

In `meta.yaml` file I've added:

```
about:
  #...
  license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE.txt'
  #...
```

Error from CircleCI:

```
OSError: Failed to copy /staged-recipes/build_artefacts/pyjc_1506951516432/work/pyjc-0.1.0/LICENSE.txt to /staged-recipes/build_artefacts/pyjc_1506951516432/_b_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placeho/info/LICENSE.txt.  Error was: Command 'cp -a /staged-recipes/build_artefacts/pyjc_1506951516432/work/pyjc-0.1.0/LICENSE.txt /staged-recipes/build_artefacts/pyjc_1506951516432/_b_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placeho/info/LICENSE.txt' returned non-zero exit status 1.
Exception ignored in: <bound method BaseFileLock.__del__ of <filelock.UnixFileLock object at 0x7fe63de08940>>
Traceback (most recent call last):
  File "/opt/conda/lib/python3.6/site-packages/filelock.py", line 305, in __del__
  File "/opt/conda/lib/python3.6/site-packages/filelock.py", line 292, in release
  File "/opt/conda/lib/python3.6/site-packages/filelock.py", line 372, in _release
TypeError: 'NoneType' object is not callable
Exited with code 1
```

If I exclude that line, the test passes.

I've tried `license_file: LICENSE.txt` previously and got similar error.

Reply:
I've also just tried removing `LICENSE.txt` file entirely, with `meta.yaml` updated to:

```
about:
  #...
   license_file: LICENSE.txt
  #...
```

still no luck.

```
OSError: Failed to copy /staged-recipes/build_artefacts/pyjc_1506952147885/work/pyjc-0.1.0/LICENSE.txt to /staged-recipes/build_artefacts/pyjc_1506952147885/_b_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placeho/info/LICENSE.txt.  Error was: Command 'cp -a /staged-recipes/build_artefacts/pyjc_1506952147885/work/pyjc-0.1.0/LICENSE.txt /staged-recipes/build_artefacts/pyjc_1506952147885/_b_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placeho/info/LICENSE.txt' returned non-zero exit status 1.
Exception ignored in: <bound method BaseFileLock.__del__ of <filelock.UnixFileLock object at 0x7fcfc6bb0a20>>
Traceback (most recent call last):
  File "/opt/conda/lib/python3.6/site-packages/filelock.py", line 305, in __del__
  File "/opt/conda/lib/python3.6/site-packages/filelock.py", line 292, in release
  File "/opt/conda/lib/python3.6/site-packages/filelock.py", line 372, in _release
TypeError: 'NoneType' object is not callable
Exited with code 1
```

Reply:
I see that my initial release `0.1.0` did not have `MANIFEST.in`. Maybe for this release `0.1.0` I will just remove `license_file: LICENSE.txt`? (and in next release `0.1.1` I will re-include that line?)

Reply:
@conda-forge/staged-recipes -- Anyone seen this type of error before? It looks like there is some issue manually packaging the license file:

https://travis-ci.org/conda-forge/staged-recipes/builds/282253129?utm_source=github_status&utm_medium=notification

Reply:
Alternatively, let me get release `0.1.1` ready (on PyPi). I will re-push to conda-forge then.

Reply:
@synapticarbors, the pypi tarball doesn't have the license file. That's the reason for the error.

Reply:
@isuruf The error was occurring when the license file was included in the PR and was being included using standard:

https://conda-forge.org/docs/meta.html#packaging-the-license-manually

The PR has been updated several times since I originally made the comment and one of the commits removed the license file.

Reply:
The error was, `subprocess.CalledProcessError: Command 'cp -a /Users/travis/miniconda/conda-bld/pyjc_1506951965038/work/pyjc-0.1.0/LICENSE.txt ` which means the `license_file` entry was looking at the `SRC_DIR` instead of `RECIPE_DIR`